clang/include/clang/Basic/DiagnosticDriverKinds.td

```diff
@@ -195,6 +195,8 @@ def err_drv_Xopenmp_target_missing_triple : Error<
   "cannot deduce implicit triple value for -Xopenmp-target, specify triple using -Xopenmp-target=<triple>">;
 def err_drv_invalid_Xopenmp_target_with_args : Error<
   "invalid -Xopenmp-target argument: '%0', options requiring arguments are unsupported">;
+def err_drv_unsupported_opt_sycl : Error<
```

Comment:
Do we really need a SYCL specific error here? Can we piggyback on existing error format (err_drv_argument_only_allowed_with)?

Thanks

Reply:
That's a good point.  There is an inverse `err_drv_argument_not_allowed_with` as well.  I will see about updating this.